packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.scss

```diff
@@ -34,6 +34,7 @@
 
     @include mobile {
         width: 100%;
+        right: 0;
```

Comment:
Why align right if it's fullscreen-wide on mobile?

Reply:
Hi @rostislav-deriv, forgot to remove it 😅